Formula/slicot.rb

```diff
@@ -0,0 +1,22 @@
+class Slicot < Formula
+  desc "Fortran 77 algorithms for computations in systems and control theory"
+  homepage "http://www.slicot.org"
+  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/s/slicot/slicot_5.0+20101122.orig.tar.gz"
```

Comment:
The "release notes" say that there is a version 5.6 dated February 5, 2017: http://slicot.org/objects/software/shared/Release.Notes

Reply:
Only 4.5 is available under GNU. The licencse has changed for 4.6 and it cannot be distributed: http://slicot.org/obtaining-slicot:

> The copyright for the SLICOT Library remains with the LICENSER. The LICENSEE has to ensure that SLICOT Library or parts of it are not distributed outside the institution of the LICENSEE.